sdk-java/src/main/java/ly/count/sdk/java/internal/SDKCore.java

```diff
@@ -713,6 +713,9 @@ private boolean processCrash(InternalConfig config, Long id) {
 
         Request request = ModuleRequests.nonSessionRequest(config);
         ModuleCrash.putCrashIntoParams(crash, request.params);
```

Comment:
Req params for crashes was missing. It was not before. After adding 'av' param, I changed the location of req param adding. And also the process for crash sending is same but it sends a different 'Ping' maybe this can be also refactored